components/modal/ModalLogin.vue

```diff
@@ -116,6 +116,28 @@ export default {
 	},
 	methods: {
 		...mapMutations('modal', [`hideModal`]),
+		errorHandling(type) {
+			if (type === 'register') {
+				if (this.name.length < 3) {
+					return this.error = 'Nickname should be at least 3 characters long.';
+				} else {
+					this.error = '';
+				}
+			}
+			if (!/\S+@\S+\.\S+/.test(this.email)) {
+				return this.error = 'This is not a valid email.';
+			} else {
+				this.error = '';
+			}
+			if (this.password.length < 6) {
+				return this.error = 'Password needs to be at least 6 characters long.';
+			} else {
+				this.error = '';
+			}
+			if (this.error === '') {
```

Comment:
Remove this block completely: `if (this.error === '') {
				type === 'register' ? this.submitRegisterForm() : this.submitForm();
			}`